lib/actions/serverSetupActions.js

```diff
@@ -402,6 +417,33 @@ export function applyServer() {
     };
 }
 
+export function applyServerAgain() {
+    return (dispatch, getState) => {
+        const portToUser = getState().app.adapter.ports
+        .get(getState().app.adapter.selectedAdapterIndex);
+        const versionInfoToUse = getState().app.adapter.versionInfos
+        .get(getState().app.adapter.selectedAdapterIndex);
+        const adapter = getState().app.adapter.adapters
+        .get(getState().app.adapter.selectedAdapterIndex);
+        dispatch(openAdapter(portToUser, versionInfoToUse))
+        .then(() => {
+            adapter.connectedDevices.forEach(device => {
+                dispatch(connectToDevice(device));
+            });
```

Comment:
Ref. our discussion yesterday we decided not to reconnect to devices. The devices may not be advertising anymore, and this could lead to errors and confusion.